tests/php/src/Admin/PolyfillsTest.php

```diff
@@ -99,4 +99,27 @@ public function test_registration() {
 
 		$this->assertTrue( wp_style_is( 'wp-components', 'registered' ) );
 	}
+
+	/**
+	 * Get inline script.
+	 *
+	 * @param string $handle Script handle.
+	 * @param string $position Script position.
+	 * @param bool   $display Whether to display the script.
+	 *
+	 * @return string
+	 */
+	public static function get_inline_script( $handle, $position = 'after', $display = false ) {
+		if ( method_exists( wp_scripts(), 'get_inline_script_tag' ) ) {
+			$script = wp_scripts()->get_inline_script_tag( $handle, $position );
+
+			if ( ! $script ) {
+				return false;
+			}
+
+			return $script;
+		} else {
+			return wp_scripts()->print_inline_script( $handle, $position, $display );
```

Comment:
Might as well remove the `$display` param altogether.

```suggestion
	 *
	 * @return string
	 */
	public static function get_inline_script( $handle, $position = 'after' ) {
		if ( method_exists( wp_scripts(), 'get_inline_script_tag' ) ) {
			$script = wp_scripts()->get_inline_script_tag( $handle, $position );

			if ( ! $script ) {
				return false;
			}

			return $script;
		} else {
			return wp_scripts()->print_inline_script( $handle, $position, false );
```